airflow_prometheus_exporter/prometheus_exporter.py

```diff
@@ -51,7 +53,7 @@ class DelayAlertMetaData(Base):
         alert_target = Column(String)
         alert_external_classification = Column(String)
         alert_report_classification = Column(String)
-        sla_interval = Column(Float)
+        sla_interval = Column(String)
```

Comment:
Changed sla_interval from float (meaning days) to string. We now can support seconds, minutes, hours, days and weeks. e.g. 1s, 2m, 3h, 4d, 5w.

Reply:
BTW, the column will be defined as `varchar(64)`